src/client/pages/OfferNew/Checkout/index.tsx

```diff
@@ -337,6 +340,10 @@ export const Checkout: React.FC<CheckoutProps> = ({
       {signStatus?.signState === SignState.Completed && (
         <CheckoutSuccessRedirect offerData={offerData} />
       )}
+      <SignFailModal
```

Comment:
I think this feedback is good for all markets, but story was only for Norway. Should I add a check for NO market or leave it for all?